modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -77,6 +166,47 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
     Events.fireTableModified(editor, table.dom, data.effect);
   };
 
+  const toggleTableClass = (_ui: boolean, requestedClass: string) => {
+    getSelectionStartCellOrCaption(editor).each((startCell) => {
+      TableLookup.table(startCell, isRoot).filter(Fun.not(isRoot)).each((table) => {
+        if (Class.has(table, requestedClass)) {
+          Class.remove(table, requestedClass);
+        } else {
+          Class.add(table, requestedClass);
+        }
+
+        Events.fireTableModified(editor, table.dom, Events.structureModified);
+      });
+    });
+  };
+
+  const toggleTableCellClass = (_ui: boolean, requestedClass: string) => {
+    getSelectionStartCell(editor).each((startCell) => {
+      TableLookup.table(startCell, isRoot).filter(Fun.not(isRoot)).each((table) => {
```

Comment:
Why do you have `.filter(Fun.not(isRoot))` here? We're not operating on the table, only the cells within so I don't believe this is needed/makes sense.